package/rmt-server.spec

```diff
@@ -27,10 +27,10 @@
 %define rmt_group   nginx
 
 Name:           rmt-server
-Version:        0.0.6
+Version:        0.0.7
 Release:        0
 Summary:        Repository mirroring tool and registration proxy for SCC
-License:        GPL-2.0+
+License:        GPL-2.0-or-later
```

Comment:
What is making this change? The RPM linter complains `Unknown license 'GPL-2.0-or-later'`